desktop/components/react/utils/renderReactLayout.js

```diff
@@ -116,8 +121,7 @@ export function renderReactLayout (options) {
         )
       }
     }
-
-    return html
+    return { html, styledComponentCSS }
```

Comment:
Thinking about this I agree; since we can technically pass any kind CSS the thing that extracts it shouldn't matter. Should probably update this in the `react_index.jade` file too.